recipes/iris-ued/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "5.0.1" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "544b62bc1e484e59a68d79fee049b9293da8df1b1ea47a558d1ce7f36f1ffa5e" %}
+
+package:
+  name: 'iris-ued'
+  version: '{{ version }}'
+
+source:
+  fn: 'iris-ued-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/i/iris-ued/iris-ued-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - iris = iris.gui:run
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy >=1.12
+    - scipy >=0.17
+    - h5py >=2.6.0
+    - pyqt >=5
+    - scikit-image >=0.13
+    - scikit-ued >=0.5.9
+    - pyqtgraph >=0.10
+    - qdarkstyle >=2.5.4
+    - npstreams >=1.5.2
+    - cached_property >=1.3
+
+test:
```

Comment:
Can you test the entry point:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

Reply:
The entry point launches a GUI, which would run forever without any human input to close it (as it is implemented right now). Any idea on how to get around this?

Entry point: `python -m iris` is equivalent to `python -c 'from iris.gui import run; run()'`

Reply:
missed that this was a gui script. you can leave out the test.